app/index.html

```diff
@@ -34,25 +32,66 @@ <h1 class="header__title visuallyhidden">
         </svg>
       </a>
       <!-- Menu -->
-      <nav id="menu" class="menu">
-        <a class="menu__link" href="#">Packages</a>
-        <a class="menu__link" href="#">Contribute</a>
-      </nav>
+      <!-- <nav id="menu" class="menu"> -->
+        <!-- Add new menu items here -->
+        <!-- <a class="menu__link" href="#">Packages</a> -->
+      <!-- </nav> -->
       <!-- Burger Button -->
-      <button id="burger-button" class="burger-button">
+      <!-- <button id="burger-button" class="burger-button">
         <svg class="burger-button__svg" viewBox="0 0 100 100">
           <use href="svg/vectors.svg#burger"></use>
         </svg>
-      </button>
+      </button> -->
     </header>
-    <div class="top-background"></div>
+    <!-- <div class="top-background"></div> -->
     <!-- Page Wrapper -->
     <main class="wrapper">
-      <h2 class="wrapper__title">
-        Welcome to the world of shiny packages created by Appsilon!
-      </h2>
       <div id="hex-grid" class="hex-grid"></div>
       <div id="wrapper-info" class="wrapper__info"></div>
+      <!-- Page Footer -->
+      <footer class="footer">
+        <a
+          href="https://github.com/Appsilon"
+          class="footer__link"
+          target = "_blank"
+          rel = "noopener noreferrer"
+        >
+          <svg class="footer__svg" viewBox="0 0 100 100">
+            <use href="svg/vectors.svg#github"></use>
+          </svg>
+        </a>
+        <a
+          href="https://www.linkedin.com/company/appsilon/"
+          class="footer__link"
+          target = "_blank"
+          rel = "noopener noreferrer"
+        >
+          <svg class="footer__svg" viewBox="0 0 100 100">
+            <use href="svg/vectors.svg#linkedin"></use>
+          </svg>
+        </a>
+        <a
+          href="https://twitter.com/appsilon"
+          class="footer__link"
+          target = "_blank"
+          rel = "noopener noreferrer"
+        >
+          <svg class="footer__svg" viewBox="0 0 100 100">
+            <use href="svg/vectors.svg#twitter"></use>
+          </svg>
+        </a>
+        <p class="footer__text">Proudly supported by</p>
```

Comment:
"Proudly supported by RStudio" is unfortunately misleading. It is a package supported by Appsilon. Is there a specific reason we want to link to RStudio? Or maybe this is just a placeholder for now and will be changing for different package providers?

Reply:
I've seen it on tidyverse website (https://www.tidyverse.org/), so I thought it would be good to show the link to rstudio, since we are a certified partner. But right, the text is a bit unfortunate in this context